runtests.py

```diff
@@ -30,6 +30,7 @@
                 "DIRS": ["bread/templates"],
             }
         ],
+        DEFAULT_AUTO_FIELD="django.db.models.AutoField",
```

Comment:
This only affects the Django app used by the unit tests, and is needed to fix some warnings.